glossary/README.md

```diff
@@ -20,26 +21,29 @@ Time window – time in which reliability of service is measured and calculated.
 Error budget – a measurement that allows us to see how much SLO can be violated in a given period of time.
 
 Occurrences - error budget calculation method. It is calculated by counting the number of good events in the total count
-of the events. Example: Count if total number of measured latencies, measured in last two weeks, was below 100 ms for 95% of
-measurements. 
+of the events. Example: Count if total number of measured latencies, measured in last two weeks, was below 100 ms for 95%
+of measurements.
 
 TimeSlices - error budget calculation method. This method uses ratio of good slices to total slices in budgeting
 period. Example: How many minutes, in last two weeks, your service had latency below 100ms in measurements in these minutes.
 
 RatioTimeSlices - error budget calculation method. It is calculated by taking average of good events to total
 events in all slices. Example: What is average percent of good events to total events in every minute for last two weeks.
 
--- Next iteration
+Ratio Metrics - is an SLI composed of a relation of two metrics. Most commonly called good (numerator) and total
+events (denominator). When picking objective for your service you specify how big part of total events should be good events.
 
-Reliability Burn Down
+Threshold Metrics - SLI where metrics returned form a query are compared to an arbitrarily chosen threshold. A comparator
+can be lt, le, gt, ge. Based on the result of a comparison is determined if the error budget is burnt or not.
 
-Ratio Metrics
+Composite SLO - it is SLO that is composed of few different objectives that can be seen as SLOs as well.
+Each objective that is part of composite SLO can have its own threshold, query and data source. Composite SLO burns its
+error budget when any of SLOs beneath burns their error budget. User should be able to specify weight that will decide
+if composite SLO is burning faster or slower depending on which inner SLO is burning its error budget.
 
-Threshold Metrics
-
-Composite SLO
-
-Aggregate SLO
+Aggregate SLO - it is SLO that aggregates different currently existing SLOs. Each SLO stays as an independent SLO but
```

Comment:
Could you elaborate what do you mean :D? I think how they are difference in calculations is shown in here, but maybe I could describe better what differences end user will see declaring both of them. Is it what you meant?

Reply:
Ideally a description of what the differences between the two approaches are, and how that impacts calculations, etc

Reply:
Ahh... I need to say that I failed here. I shoudn't have add this one in here. I have seen some mentions about concept of aggregate slo in few places and it was described as I wrote in here. But I agree it is kinda new concept and I don't have enough knowledge about it to define precisely enough aggregate SLO. I think as I cannot find any provider that tries to implement that one and it isn't commonly used term I can remove that one and wait until it will be easier to describe. It just isn't `standard term`, sorry for that one.